src/qt/test/apptests.cpp

```diff
@@ -84,8 +84,11 @@ void AppTests::appTests()
     // Reset global state to avoid interfering with later tests.
     LogInstance().DisconnectTestLogger();
     AbortShutdown();
-    UnloadBlockIndex(/* mempool */ nullptr);
-    WITH_LOCK(::cs_main, g_chainman.Reset());
+    {
+        LOCK(cs_main);
+        UnloadBlockIndex(/* mempool */ nullptr, g_chainman);
```

Comment:
Resolved in 74f73c783d46b012f375d819e2cd09c792820cd5 (I just removed that comment since "mostly pure refactor" isn't too meaningful :smile: 

For the `[META]` tags, I literally just came up with it randomly, mostly to describe high-level things that apply to the entire commit or things that reference other commits.